engine/apps/alerts/tasks/distribute_alert.py

```diff
@@ -26,6 +26,7 @@ def distribute_alert(alert_id):
     if alert.is_the_first_alert_in_group:
         alert_group = AlertGroup.all_objects.filter(pk=alert.group_id).get()
         alert_group.start_escalation_if_needed(countdown=TASK_DELAY_SECONDS)
+        alert_group_escalation_snapshot_built.send(sender=distribute_alert, alert_group=alert_group.id)
```

Comment:
I think you need to pass the `alert_group` instance here, not the id (unless you change this to be a task)